arches_rdm/templates/views/components/etl_modules/migrate-rdm.htm

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+
+{% block task_details %}
+<!-- ko ifnot: loading() -->
```

Comment:
It seems this task details block is leaking into the task status area:

<img width="1060" alt="Screenshot 2024-02-08 at 6 12 48 PM" src="https://github.com/archesproject/arches-rdm/assets/38668450/c04f4ef2-54ab-4679-ae2b-c41c2e1c5b0f">